experimental/c-cpp/manager.py

```diff
@@ -1230,6 +1235,24 @@ def load_introspector_report():
   return summary_report
 
 
+def get_heuristics_to_use() -> List[Type[FuzzHeuristicGeneratorBase]]:
+  """Returns the list of possible heuristics to use for harness generation."""
+  heuristics_to_use = os.getenv('GENERATOR_HEURISTICS', 'all')
+  heuristics_to_apply = []
+  all_possible_heuristics = [
+      FuzzerGenHeuristic6, FuzzerGenHeuristic5, FuzzerGenHeuristic4,
+      FuzzerGenHeuristic3, FuzzerGenHeuristic2, FuzzerGenHeuristic1
+  ]
+  if heuristics_to_use == 'all':
+    heuristics_to_apply = all_possible_heuristics
+  else:
+    possible_heuristics = set(heuristics_to_use.split(','))
+    for heuristic in all_possible_heuristics:
+      if heuristic.name in possible_heuristics:
+        heuristics_to_apply.append(heuristic)
```

Comment:
Just out of curiosity: Does the order in all_possible_heuristics reflect priority? E.g., heuristic 6 has a higher/lower priority than 5.

Reply:
No, the numbering is purely for ID purposes.